static/app/components/modals/widgetBuilder/addToDashboardModal.tsx

```diff
@@ -177,16 +179,16 @@ function AddToDashboardModal({
         <StyledButtonBar gap={1.5}>
           <Button
             onClick={handleAddAndStayInDiscover}
-            disabled={!canSubmit}
-            title={canSubmit ? undefined : t('Select a dashboard')}
+            disabled={!canSubmit || selectedDashboardId === NEW_DASHBOARD_ID}
+            title={canSubmit ? undefined : SELECT_DASHBOARD_MESSAGE}
```

Comment:
I think we should also add a tooltip when we disable this for the new dashboard case. Maybe "Select a saved dashboard to add to it"?

Reply:
Yes. I think it's a good idea as it will make it clear to the user why the button is disabled

Reply:
I'll make a follow up PR with a tooltip change since it's not critical for disabling!